Flaky drawer e2e test case

# Bug Report

The newly introduced drawer e2e test for overlay scroll behaviour is flaky. Investigate how to make it stable.

Still flaky on master:

https://circleci.com/gh/dynatrace-oss/barista/8838

```
Drawer
 ✓ should open drawer on button click
 ✓ should close drawer on button click
 ✓ should close chart tooltip on scroll
 ✓ should close overlay on scroll
 ✓ should move heatfield overlay with the chart on scroll
✖ should move selection area overlay with the chart on scroll (screenshots:
 /home/circleci/barista/dist/components-e2e/screenshots/test-38-errors/1_1.png)

   1) AssertionError: expected false to be truthy

      Browser: Edge 18.18363.0 / Windows 10.0.0

   (https://automate.browserstack.com/builds/34f08ef4451bb67f20194193589aae1cceeb6e74/sessions/02748d95da516a16516abb80fc4bd6a1407d8d6b)
      Screenshot:

   /home/circleci/barista/dist/components-e2e/screenshots/test-38-errors/1_1.png

         122 |  await testController.wait(500);
         123 |  const currPosition = await
      rangeOverlay.getBoundingClientRectProperty('top');
         124 |  await
      testController.expect(prevPosition).notEql(currPosition);
         125 |  await testController
         126 |    .expect(isCloseTo(currPosition, prevPosition -
      scrollDistance, 10))
       > 127 |    .ok();
         128 |
         129 |  await closeOverlay();
         130 |
         131 |  await testController.expect(await
      selection.exists).notOk();
         132 |  await testController.expect(await overlay.exists).notOk();

         at Object.<anonymous>

   (/home/circleci/barista/apps/components-e2e/src/components/drawer/drawer.e2e.ts:127:6)
         at fulfilled
      (/home/circleci/barista/apps/components-e2e/src/components/drawer/drawer.e2e.ts:4:58)

```

I think we should refactor it to the chaining approach and reduce test speed as I've done in this PR: https://github.com/dynatrace-oss/barista/pull/474

--- a/apps/components-e2e/src/components/drawer/drawer.e2e.ts
+++ b/apps/components-e2e/src/components/drawer/drawer.e2e.ts
@@ -146,7 +146,7 @@ test('should move selection area overlay with the chart on scroll', async (testC
     .notEql(currPosition)
     .expect(isCloseTo(currPosition, prevPosition - scrollDistance, 10))
     .ok()
-    .click(closeButton)
+    .click(closeButton, { speed: 0.3 })
     .expect(selection.exists)
     .notOk()
     .expect(overlay.exists)

